census_blueprints/execute_sync.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_BAD_REQUEST = 201
+EXIT_CODE_SYNC_REFRESH_ERROR = 210
+EXIT_CODE_UNKNOWN_ERROR = 211
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--sync-id', dest='sync_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def execute_sync(sync_id, access_token):
+    """
+    Executes/starts a Census Sync
+    """
+    sync_post_api = f"https://bearer:{access_token}@app.getcensus.com/api/v1/syncs/{sync_id}/trigger"
+    api_headers = {
+        'Content-Type' : 'application/json'
+        }
+    try:
+        sync_trigger_response = requests.post(sync_post_api, headers=api_headers)
+        # check if successful, if not return error message
+        if sync_trigger_response.status_code in [200, 201]:
+            sync_trigger_json = sync_trigger_response.json()
+        else:
+            print(f"Sync request failed. Reason: {sync_trigger_response.content}")
+            sys.exit(EXIT_CODE_BAD_REQUEST)
+    except Exception as e:
+        print(f"Sync trigger request failed due to: {e}")
+        sys.exit(EXIT_CODE_BAD_REQUEST)
+
+    if sync_trigger_json['status'] == 'success':
+        print("Successfully triggered sync")
+        return sync_trigger_response.json()
+
+    elif sync_trigger_json['status'] == 'error':
```

Comment:
Update `elif` to be just an `if` because the return in the above if means the else never executes. It's a common short circuit type convention that we should follow moving forward.